src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -37,30 +37,84 @@ import scala.annotation.tailrec
  */
 
 object Homework :
-
+  //===========================================================================
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean =
+      if (b)
+        false
+      else
+        true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    end not
+    //===========================================================================
 
-    def or(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean =

```

Comment:
The solution could be simplified, 2 `if` branches instead of 3. Suppose `left` is `true`. What value determines the result value of the expression?